example/standalone_git/README.md

```diff
@@ -3,20 +3,69 @@
 Before running the example, these variables must be set beforehand.
 
-| Variable   | Default value | Description |
-| ------------- | ------------- | ------------- |
-| `git_remote_url`  | " "  | URL of the remote git repository |
-| `git_checkout_branch`  | " "  |Branch to checkout and track.|
-| `git_flow_storage_directory`  | /opt/nifi-registry/flow-storage  | Default value is set by image, original default value was "./flow-storage". This variable is used in clone script as a destination directory for clone. It is also used in NiFi Registry configuration.|
-| `git_remote_to_push`  | origin  |This variable is used in clone script to set origin name using `-o, --origin <name> use <name> instead of 'origin' to track upstream`. It is also used in NiFi Registry configuration.|
-| `git_access_user`  | " "  |Username|
-| `git_access_password`  | " "  |Password|
-| `git_user_name`  | nifi-registry  |Developer|
-| `git_user_email`  | nifi-registry@localhost  |Developer email|
-
-__NOTE!__ In this version of the example, the variables above __must be written directly in the [main.tf](main.tf#L52-L59)__.
-We are still improving this workflow to be more secure and agile.
+| Variable   |  Default value |Defined from CLI | Description |
+| ------------- | ------------- | ------------- | ------------- |
+| `git_remote_url`  | " "  | Yes |URL of the remote git repository. Use HTTPS. |
+| `git_checkout_branch`  | " "  | Yes| Branch to checkout and track.|
+| `git_flow_storage_directory`  | /opt/nifi-registry/flow-storage  | No| Default value is set by image, original default value was "./flow-storage". This variable is used in clone script as a destination directory for clone. It is also used in NiFi Registry configuration.|
+| `git_remote_to_push`  | origin  | No |This variable is used in clone script to set origin name using `-o, --origin <name> use <name> instead of 'origin' to track upstream`. It is also used in NiFi Registry configuration.|
+| `git_access_user`  | " "  | Yes | GitHub Username |
+| `git_access_password`  | " "  | Yes | [Personal token from GitHub](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)|
+| `git_user_name`  | nifi-registry  | No | Developer|
+| `git_user_email`  | nifi-registry@localhost  | No | Developers email|
 
+The variables marked with `Yes` in the column `Defined from CLI` must be pass as arguments from the command line.
+The variable names have been abbreviated, such as:
+
+| Variable | Abbreviated parameter <br>name on CLI |
+| ------------- | ------------- | 
+|`git_remote_url` | `repo` |
+|`git_checkout_branch`  | `branch` |
+|`git_access_user`  | `user` |
+|`git_access_password`  | `token` |
+
+Example with description of the parameters:
+```bash
+make up repo=<GitHub-repository> branch=<branch to checkout and track> user=<GitHub username> token=<personal token from GitHub> 
```

Comment:
Make up is also working without asking parameter ,all these parameter are optional or compulsory?

Reply:
Are you sure you are running from the right branch? `issue_20_manage_secrets`

Reply:
I am working on this branch issue_50_manage_secrets

Reply:
now its working